ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -51,6 +51,12 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: ipi-aws-ovn-hybrid-custom-vxlan-port
+- as: vsphere-e2e-vsphere
```

Comment:
Following the other naming convention shouldn't this be `vsphere-e2e-operator`?

Reply:
+1